test/batchnorm.jl

```diff
@@ -5,4 +5,11 @@
         NNlibCUDA.batchnorm(v, v, m, v, v, 1.0; training=training)
         NNlibCUDA.∇batchnorm(v, v, m, m, v, v, 1.0; training=training)
     end
+
+    @testset "track_stats=false" begin
+        for training in (false, true)
+            NNlibCUDA.batchnorm(v, v, m, nothing, nothing, 1.0; training=training)
+            NNlibCUDA.∇batchnorm(v, v, m, m, nothing, nothing, 1.0; training=training)
```

Comment:
I think this is what's causing the CI failures? `g` and `b` are meant to be args 1 and 2, no?